src/main/java/com/nikhil/xurl/shortninglogic/XurlImpl.java

```diff
@@ -87,19 +92,27 @@ public String getLongUrl(String shortUrlHash) {
 	}
 
 	/**
-	 * Yet to implement
+	 * {@inheritDoc}
 	 */
 	@Override
 	public Integer getHitCount(String longUrl) {
 		return urlRepo.findById(longUrl).get().getHitCount();
 	}
 
-	/**
-	 * Yet to implement
+	
```

Comment:
The Java docs in the Interface don't match with the implementation. Do the changes accordingly. Either change Java docs or change the implementation.

Reply:
Changed the java doc.